packages/library/src/providers/CFDProvider/CFDProvider.tsx

```diff
@@ -11,12 +11,14 @@ type TCFDState = {
 type TCFDContext = {
     getCFDState: <T extends keyof TCFDState>(key: T) => TCFDState[T];
     setCfdState: <T extends keyof TCFDState>(key: T, value: TCFDState[T]) => void;
+    hide: () => void;
+    show: (ModalContent: ReactNode) => void;
```

Comment:
Why are we having show and hide in CFDProvider, completely defeats the purpose of ModalProvider